src/styles.scss

```diff
@@ -27,45 +17,50 @@ $elevation-3: 0 1px 8px 0 rgba(0, 0, 0, 0.2), 0 3px 4px 0 rgba(0, 0, 0, 0.14),
 $size-0: 3.25rem;
 $size-8: 0.5rem;
 
-// Model settings
-$modal-background-background-color: $secondary;
-$modal-card-head-radius: 0;
-$modal-card-foot-radius: 0;
-$modal-card-head-padding: 20px 20px 0;
-$modal-card-head-border-bottom: 0;
-$modal-card-head-background-color: $white;
-$modal-card-foot-border-top: 0;
-$modal-card-foot-background-color: $white;
-
-$slider-thumb-background: $primary;
-$slider-thumb-border: 1px solid $primary;
-
-// Titles
-$title-color: $black;
-$title-weight: 200;
-
-// Labels
-$label-weight: $weight-light;
+//
+// Bulma Variables
+//
+@import 'bulma/sass/utilities/_all';
 
-// Inputs
-$input-shadow: none;
-$input-focus-box-shadow-size: none;
+//
+// Our Variables
+//
+$mobile: $tablet - 2px;
+$secondary: $grey-lighter;
 
 //
 // Bulma Styles
 //
-@import 'bulma/sass/utilities/_all';
+@import 'bulma/sass/base/_all';
```

Comment:
one of the previous tickets was to remove this, right? i take it the re-ordering solves the problem and thats why base is back ?

Reply:
Leaving this here as removing it makes a bunch of things just look stupid. i.e. anyone that uses this lib will need to include it themselves. Going to handle the use case in the Forms CDN lib using this method:

```scss
.oneblink-apps-react-styles {
  box-sizing: border-box;

  @import '~@oneblink/apps-react/dist/styles.scss';
}
```

As the only styles added to `html` and `body` are most likely already going to be set in the consuming application.

Reply:
sweet, makes sense :)